app/assets/javascripts/graphs.js

```diff
@@ -1,48 +1,37 @@
-var title = 'cpp2';
-console.log(title);
+$(document).ready(function() {
```

Comment:
Changing line 1 to

```
$(document).on('turbolinks:load', function() {
```

worked for me on an instant load. as per these [rails guidelines](http://guides.rubyonrails.org/working_with_javascript_in_rails.html#turbolinks).

Reply:
I think basically it worked on reload because the browser then has a cached copy, which isn't available on first load - but this seems to work nicely.